src/extensions/score_metamodel/checks/check_options.py

```diff
@@ -64,31 +89,18 @@ def remove_prefix(word: str, prefixes: list[str]) -> str:
                     need, f"is missing required {field_type}: `{field}`."
                 )
             continue  # Skip empty optional fields
-
-        values: list[str]
-
-        if isinstance(raw_value, str):
-            values = [raw_value]
-        elif isinstance(raw_value, list) and all(isinstance(v, str) for v in raw_value):
-            values = raw_value
-        else:
-            values = [str(raw_value)]
-
+        try:
+            values = _normalize_values(raw_value)
+        except ValueError as err:
+            raise ValueError(
+                f"An Attribute inside need {need['id']} is "
+                "not of type str. Only Strings are allowed"
+            ) from err
```

Comment:
Nit (fix in next PR): 

Add comment explaining why we did this. 


```suggestion
       # Try except used to add more context to Error without passing variables just for that to function
        try:
            values = _normalize_values(raw_value)
        except ValueError as err:
            raise ValueError(
                f"An Attribute inside need {need['id']} is "
                "not of type str. Only Strings are allowed"
            ) from err
```